openbas-api/src/main/java/io/openbas/service/InjectExpectationService.java

```diff
@@ -644,6 +645,20 @@ public List<InjectExpectation> expectationsForAssets(
 
   // -- PREVENTION --
 
+  public List<InjectExpectation> preventionExpectationsNotExpired(final Integer expirationTime) {
+    return this.injectExpectationRepository.findAll(
+        Specification.where(
+            InjectExpectationSpecification.type(PREVENTION)
+                .and(
+                    InjectExpectationSpecification.agentNotNull()
+                        .and(
+                            InjectExpectationSpecification.assetNotNull()
+                                .and(
+                                    InjectExpectationSpecification.from(
+                                        Instant.now()
+                                            .minus(expirationTime, ChronoUnit.MINUTES)))))));
+  }
```

Comment:
```suggestion
  public List<InjectExpectation> manualExpectationsNotExpired(final Integer expirationTime) {
    return this.injectExpectationRepository.findAll(
        Specification.where(
            InjectExpectationSpecification.type(MANUAL)
                .and(
                    InjectExpectationSpecification.agentNotNull())
                .and(
                    InjectExpectationSpecification.assetNotNull())
                .and(
                    InjectExpectationSpecification.from(
                        Instant.now()
                            .minus(expirationTime, ChronoUnit.MINUTES)))));
  }
```

I don't have test it but it's easier to read